misc/deployment/variables.yml

```diff
@@ -5,3 +5,5 @@ variables:
   value: 'b2b-global-api'
 - name: b2bNewsApiImageRepository
```

Comment:
**suggestion:** please move service variables to variables.yml files in service directories. We don't want to have references to services in global variables.yml.